pydarn/exceptions/plot_exceptions.py

```diff
@@ -45,8 +45,8 @@ class NoDataFoundError(Exception):
     This error is raised when no data is found for
     the given beam and parameter
     """
-    def __init__(self, parameter: str, beam_num: int, start_time: int,
-                 end_time: datetime.datetime, opt_beam_num: int):
+    def __init__(self, parameter: str, beam_num: int, opt_beam_num: int,
+                 start_time: int = None, end_time: datetime.datetime = None):
```

Comment:
I think this is the wrong type for start_time

```suggestion
                 start_time: datetime.datetime = None,
                 end_time: datetime.datetime = None):
```